src/test/resources/com/cognifide/gradle/aem/test/debug/build.gradle

```diff
@@ -2,11 +2,11 @@ buildscript {
     repositories {
         jcenter()
         mavenLocal()
-        maven { url  "http://dl.bintray.com/cognifide/maven-public" }
```

Comment:
Since you are changing it anyway: I'd wrap any non-template string with single quotes instead of double quotes - or at least keep consistent quotation marks within build files.

Reply:
I am trying to keep it  consistent, trust me ;) but it is not most important... I am doing a lot of changes and it is quite hard to track on such detail.